README.md

```diff
@@ -209,9 +217,9 @@ Create a new git commit/tag in the process of publishing to npm.
 $ lerna publish --npm-tag=next
 ```
 
-Publish to npm with the given npm [dist-tag](https://docs.npmjs.com/cli/dist-tag) (defaults to `latest`).
+When run with this flag, `publish` will publish to npm with the given npm [dist-tag](https://docs.npmjs.com/cli/dist-tag) (defaults to `latest`).
```

Comment:
Let me know if this is too redundant. I like having this in each flag description because it makes it really clear this is for use with `publish` to someone who is looking up a command and only looks at this one flag descriptions